src/unstable-temp/DownloadImageModal/FormModel.tsx

```diff
@@ -11,6 +12,8 @@ import { PasswordInput } from './PasswordInput';
 import styled from 'styled-components';
 import { DeviceTypeOptions, DeviceTypeOptionsGroup } from './models';
 
+const DEFAULT_FORMAT = 'yyyy[-]MM[-]DD';
```

Comment:
`DATE_FORMAT` would probably be more appropriate here. `DEFAULT_FORMAT` does not tell me what it is a format for, and I also do not understand what `DEFAULT` is in reference to considering that there are no other format options available to choose from.